core/src/main/java/cn/edu/tsinghua/iot/benchmark/entity/Batch/IBatch.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package cn.edu.tsinghua.iot.benchmark.entity.Batch;
+
+import cn.edu.tsinghua.iot.benchmark.entity.Record;
+import cn.edu.tsinghua.iot.benchmark.schema.schemaImpl.DeviceSchema;
+
+import java.util.List;
+
+public interface IBatch {
+  void add(long timestamp, List<Object> values);
+
+  long pointNum();
+
+  DeviceSchema getDeviceSchema();
+
+  void setDeviceSchema(DeviceSchema deviceSchema);
+
+  void setColIndex(int colIndex);
+
+  int getColIndex();
+
+  List<Record> getRecords();
+
+  boolean hasNext();
+
+  void next();
+
+  default void finishCheck() throws Exception {
+    if (hasNext()) {
+      throw new Exception("batch should have been consumed, but it hasn't. check your code.");
+    }
+  }
+
+  void reset();
```

Comment:
I think we need to add comments on all methods in the interface. The comments contain the explanation of parameters and return value, and the function of this method

Reply:
The reason for not writing comments is that `IBatch` is not a well designed interface, but rather a temporary transitional solution, and there will be no new classes to implement `IBatch`. 
Besides, in the future, `IBatch` should be completely replaced with `MultiDeviceBatch`.

Reply:
comments added.